apps/pie-storybook/stories/testing/pie-notification.test.stories.ts

```diff
@@ -114,12 +121,18 @@ const notificationStoryMeta: NotificationStoryMeta = {
             },
         },
         leadingAction: {
-            description: 'The leading action configuration for the notification.',
+            description: `The leading action configuration for the notification. <br/> The size property can be ${actionSizes.join(', ')}.`,
             control: 'object',
+            defaultValue: {
+                summary: JSON.stringify(defaultProps.leadingAction),
+            },
         },
         supportingAction: {
-            description: 'The supporting action configuration for the notification. Appears only if `leadingAction` is provided.',
+            description: `The supporting action configuration for the notification. Appears only if leadingAction is provided. <br/>The size property can be ${actionSizes.join(', ')}.`,
```

Comment:
Just going to keep saying - these descriptions aren't very useful for automated test stories in my opinion :P